src/components/ChatApp/MessageListItem/MessageListItem.react.js

```diff
@@ -248,6 +248,45 @@ class MessageListItem extends React.Component {
               );
               break
             }
+            case 'video_play': {
+              let identifierType = data.answers[0].actions[index].identifier_type;
+              let identifier = data.answers[0].actions[index].identifier;
+              let src = `https://www.${identifierType}.com/embed/${identifier}`;
```

Comment:
You can do something like : 
```javascript
let src = `https://www.${identifierType}.com/embed/${identifier}?autoplay=1`;
```

This should work in most popular Video Hosting Websites.

Reply:
@duttaditya18 please see older comments first and give your views accordingly. Do you have any solution for the above bug? Thanks. :-)

Reply:
@PrP-11 I think you can identify if you are fetching from the History API or not. If you are, then using a certain configuration for the `src` attribute to not add the autoplay query will solve the case.